tasks/main.yml

```diff
@@ -475,10 +430,194 @@
         __mssql_conf_setting: "network forceencryption"
         __mssql_conf_setting_value: "{{ '1' if mssql_tls_enable else 'unset' }}"
 
+- name: Configure availability group on the primary node
+  when:
+    - mssql_ha_configure is defined
+    - mssql_ha_configure | bool
+    - mssql_ha_replica_type == 'primary'
```

Comment:
so in the inventory for this host you would have a host var `mssql_ha_replica_type: primary`, and for the replica hosts, you would have host vars for `mssql_ha_replica_type: replica` ?